oep-template.rst

```diff
@@ -85,32 +77,20 @@ An OEP that introduces backward incompatibilities must describe the
 incompatibilities, with their severity and an explanation of how you propose to
 address these incompatibilities.
 
-OEP submissions that do not consider backward compatibility issues are unlikely
-to be accepted.
-
-
 Reference Implementation
 ========================
 
 The reference implementation must be completed before any OEP is given "Final"
-status, but it need not be completed before the OEP is accepted. While there is
+status, but it need not be completed before the OEP is "Accepted". While there is
```

Comment:
Not to nit-pick, but I found `accepted` to read better than `"Accepted"`.

Reply:
Feel free to nitpick.  I had made the change to make it consistent with `"Final"` in the 1st phrase.  But I can easily revert the change - as readability was the goal for the consistency.

Reply:
reverted.